examples/python/RVC4/GPUStereo/gpu_stereo_gui.py

```
@@ -0,0 +1,2273 @@
+            self.pipeline.stop()
+        except Exception:
+            pass
+
+    def run(self) -> None:
+        with self.pipeline:
+            self.pipeline.start()
+            while not self._stop.is_set() and self.pipeline.isRunning():
+                try:
+                    disp = self.disp_q.get(timeout=0.2)
+                except BaseException:
+                    continue
```

Ruff (0.15.12)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)